internal/generate/tagstests/main.go

```diff
@@ -198,7 +270,7 @@ var testTfTmpl string
 
 // Annotation processing.
 var (
-	annotation = regexache.MustCompile(`^//\s*@([0-9A-Za-z]+)(\((.*)\))?\s*$`)
+	annotation = regexp.MustCompile(`^//\s*@([0-9A-Za-z]+)(\((.*)\))?\s*$`) // nosemgrep:ci.calling-regexp.MustCompile-directly
```

Comment:
is there a difference with using `regexp` vs `regexp2` for this call? Both are imported so I'm curious

Reply:
`regexp2` is needed for negative lookahead, but the API takes a few more parameters for most calls than the standard `regexp` so I used `regexp` for most cases. Maybe overly complicated